server/utils/errorMessages.ts

```diff
@@ -15,6 +15,9 @@ export default {
     invalidDate: 'The date by which the service needs to be completed must be a real date',
     mustBeInFuture: 'The date by which the service needs to be completed must be in the future',
   },
+  serviceCategories: {
+    empty: 'Select service types',
```

Comment:
i think this error could be more descriptive. e.g. "at least one service category must be selected"